### ISSUE TITLE
Feature request: staged runs to automatically use latest operation as input to next

### ISSUE THREAD

**Issue by FelixAbrahamsson** [guildai/guildai#78]
I really like the workflow I'm able to use in the latest version of guild with
```
guild run ... --stage
guild run ... --stage
guild run queue
```
However if at some point one of the staged runs requires as input the model trained in the previously staged run, I have to specify the hash of the run explicitly (as `guild run model=<hash> --stage`). Without staging, the default behavior is to use the latest run as input. Wouldn't it make sense for this to be the default behavior with staged runs as well?

**Comment by gar1t** [comment]
I create an issue doc for this: https://github.com/guildai/issues/tree/master/78-staged-runs-to-automatically-use-latest-operation

Marking as bug because this behavior is not only a good idea, it's the only sensible behavior! :)

**Comment by gar1t** [comment]
This is fixed - I believe in rc9 if not certainly in the upcoming rc10.

The steps to verify are in the issue resolution doc linked above.

**Comment by gar1t** [comment]
Fix is now available in 0.7.0.
